### PR TITLE
fix(TDI-33080): Ignore internal errors on logging via LogCatcher

### PR DESCRIPTION
**Pull request by iistomin-gl**
Ignore internal errors on logging via LogCatcher

JIRA issue: https://jira.talendforge.org/browse/TDI-33080

### CODE REVIEW

File: main/plugins/org.talend.designer.components.localprovider/components/tWarn/tWarn_java.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <HEADER
     PLATEFORM="ALL"
        SERIAL=""
-      VERSION="0.101"
+      VERSION="0.102"
```
Comment (gonchar-ivan) [inline main/plugins/org.talend.designer.components.localprovider/components/tWarn/tWarn_java.xml]: Do you know what this version for? (I don't know)

File: main/plugins/org.talend.designer.components.localprovider/components/tLogCatcher/tLogCatcher_messages.properties [hunk 2]
```diff
@@ -6,5 +6,3 @@ CATCH_JAVA_EXCEPTION.NAME=Catch Java Exception
 CATCH_TDIE.NAME=Catch tDie
 CATCH_TWARN.NAME=Catch tWarn
 CATCH_TACTIONFAILURE.NAME=Catch tActionFailure
-IGNORE_LOG_CATCHER_INTERNAL_ERRORS.NAME=Ignore LogCatcher internal errors
```
Comment (chmyga) [inline main/plugins/org.talend.designer.components.localprovider/components/tLogCatcher/tLogCatcher_messages.properties]: Why was this part removed? Do we want to always ignore the tLogCatcher internal errors?

File: main/plugins/org.talend.designer.components.localprovider/components/tWarn/tWarn_main.javajet [hunk 3]
```diff
@@ -47,7 +50,11 @@ if(priority.equals("1")){
...
 			}
 %>
-globalMap.put("<%=cid %>_WARN_MESSAGES", <%=ElementParameterParser.getValue(node, "__MESSAGE__")%>); 
-globalMap.put("<%=cid %>_WARN_PRIORITY", <%=ElementParameterParser.getValue(node, "__PRIORITY__") %>);
-globalMap.put("<%=cid %>_WARN_CODE", <%=ElementParameterParser.getValue(node, "__CODE__") %>);
+	globalMap.put("<%=cid %>_WARN_MESSAGES", <%=ElementParameterParser.getValue(node, "__MESSAGE__")%>); 
+	globalMap.put("<%=cid %>_WARN_PRIORITY", <%=ElementParameterParser.getValue(node, "__PRIORITY__") %>);
+	globalMap.put("<%=cid %>_WARN_CODE", <%=ElementParameterParser.getValue(node, "__CODE__") %>);
+	
+} catch (Exception | Error e_<%=cid%>) {
```
Comment (chmyga) [inline main/plugins/org.talend.designer.components.localprovider/components/tWarn/tWarn_main.javajet]: Do we really want to catch Errors? what if an error is an OOM? We might have huge problems with it, as it will influence the JVM, and it might not work anymore, so the job will stay in some hanged state.

Comment (iistomin-gl) [inline main/plugins/org.talend.designer.components.localprovider/components/tWarn/tWarn_main.javajet]: Agree. Errors (like OOM) should not be ignored.
